src/_experimental.scss

```diff
@@ -159,3 +165,103 @@
 		}
 	}
 }
+
+
+.provisional {
+	// Removes styling from the current WET4 Alert
+	.alert-info > :first-child::before,
+	.alert-success > :first-child::before,
+	.alert-warning > :first-child::before,
+	.alert-danger > :first-child::before {
+		content: none;
+	}
+
+	.alert {
+		background-clip: content-box;
+		background-color: inherit;
+		border-left: 6px solid #000;
+		margin-bottom: 23px;
+		margin-left: 10px;
+		padding: 0 0 0 15px;
+
+		> {
+			p, .ul, .ol {
+				margin-bottom: 0px;
+			}
+
+			* {
+				margin-left: 15px;
+			}
+		}
+
+		details {
+			margin-left: .5em;
+			padding-top: 15px;
+		}
+
+		> {
+			:first-child:not(details) {
+				margin-top: auto;
+				padding-top: 15px;
+			}
+
+			:last-child {
+				padding-bottom: 25px;
+			}
+		}
+
+		&:before {
+			font-family: "Glyphicons Halflings";
+			font-size: 26px;
+			line-height: 2.3em;
+			margin-left: -1.27em;
+			padding: 2px;
+			position: absolute;
+		}
+
+		> :first-child {
+			margin-left: 15px;
+		}
+	}
+
+	.alert-info {
+		border-color: $provisional-info-color;
+		border-image: linear-gradient(to bottom, $provisional-info-color 16px, $provisional-info-color 16px, rgba(0, 0, 0, 0) 16px, rgba(0, 0, 0, 0) 48px, $provisional-info-color 48px, $provisional-info-color 48px) 1 100%;
+
+
+		&::before {
+			color: $provisional-info-color;
+			content: "\e086";
+		}
+	}
+
+	.alert-success {
+		border-color: $provisional-success-color;
+		border-image: linear-gradient(to bottom, $provisional-success-color 16px, $provisional-success-color 16px, rgba(0, 0, 0, 0) 16px, rgba(0, 0, 0, 0) 48px, $provisional-success-color 48px, $provisional-success-color 48px) 1 100%;
+
+		&::before {
+			color: $provisional-success-color;
+			content: "\e084";
+		}
+	}
+
+	.alert-warning {
+		border-color: $provisional-warning-color;
+		border-image: linear-gradient(to bottom, $provisional-warning-color 16px, $provisional-warning-color 16px, rgba(0, 0, 0, 0) 16px, rgba(0, 0, 0, 0) 48px, $provisional-warning-color 48px, $provisional-warning-color 48px) 1 100%;
+
+		&::before {
+			color: $provisional-warning-color;
+			content: "\e107";
+		}
+	}
+
+	.alert-danger {
+		border-color: $provisional-danger-color;
+		border-image: linear-gradient(to bottom, $provisional-danger-color 16px, $provisional-danger-color 16px, rgba(0, 0, 0, 0) 16px, rgba(0, 0, 0, 0) 48px, $provisional-danger-color 48px, $provisional-danger-color 48px) 1 100%;
```

Comment:
This would be a nice to have, it will be to try to remove CSS code duplication.

SASS already have a few feature to help to avoid code duplication like
* `@ function` ref.https://sass-lang.com/documentation/at-rules/function
* The share plugin has something similar too. ref. https://github.com/wet-boew/wet-boew/blob/master/src/plugins/share/_base.scss#L7

Reply:
Addressed in #1650. Went with a mixin.